src/util/report.py

```diff
@@ -286,6 +286,21 @@ def ref(xid):
     except Exception:
         _logger.warning("Cannot announce message", exc_info=True)
 
+    # Chat window with the report will be open post-upgrade for the admin
+    if version_gte("9.0"):
```

Comment:
We can get rid of one level of `if`s in favor of a dictionary with versions, but I thought it's going to be less readable.

Reply:
You can get rid of this because we do not support upgrading to 9.0 already :)